### PR TITLE
Update YugabyteDB Managed to YugabyteDB Aeon

### PR DESCRIPTION
**Pull request by posriniv**

### CODE REVIEW

File: docs/ybm.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ ybm [flags]
       --config string      config file (default is $HOME/.ybm-cli.yaml)
       --debug              Use debug mode, same as --logLevel debug
   -h, --help               help for ybm
+      --host string        YBM Api hostname
```
Comment (ddhodge) [inline docs/ybm.md:20]: uppercase YBM refers to YugabyteDB Managed, lowercase ybm refers to the cli binary. So in this case, what are you referring to? All instances of 'YBM' should be replaced with YugabyteDB Aeon.
What is the YBM Api hostname? What is it for?
Api - > API

File: docs/ybm_api-key.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ ybm api-key [flags]
   -a, --apiKey string      YBM Api Key
```
Comment (ddhodge) [inline docs/ybm_api-key.md:22]:
```suggestion
  -a, --apiKey string      YugabyteDB Aeon account API key
```

File: docs/ybm.md (hunk 1 above)
Comment (posriniv) [reply docs/ybm.md]: `host` has now been removed. All occurrences of `YBM` in the docs have been replaced with YugabyteDB Aeon.

**Comment by posriniv** [comment]
@ddhodge can you please take another look?

**Pull request merged**